src/app/shared/services/api/directive.repo.ts

```diff
@@ -1,30 +1,46 @@
 import {
   AccountVO,
   ArchiveVO,
-  Directive,
+  DirectiveData,
   DirectiveCreateRequest,
   LegacyContact,
+  Directive,
 } from '@models/index';
 import { BaseRepo } from './base';
+import { getFirst } from '../http-v2/http-v2.service';
 
 export class DirectiveRepo extends BaseRepo {
   public async get(archive: ArchiveVO): Promise<Directive> {
-    console.warn('Directive API is currently unimplemented.');
-    return {} as Directive;
+    return getFirst(
+      this.httpV2.get(
+        `/v2/directive/archive/${archive.archiveId}`,
+        {},
+        Directive
+      )
+    ).toPromise();
   }
 
   public async getLegacyContact(account: AccountVO): Promise<LegacyContact> {
-    console.warn('Directive API is currently unimplemented.');
+    console.warn('Legacy Contact API is currently unimplemented.');
     return {} as LegacyContact;
   }
 
   public async create(directive: DirectiveCreateRequest): Promise<Directive> {
-    console.warn('Directive API is currently unimplemented.');
-    return {} as Directive;
+    return getFirst(
+      this.httpV2.post('/v2/directive', directive, Directive)
```

Comment:
I'm assuming this is just because I didn't have everything set up locally as I should, but I wasn't sure how to pass the authorization header to `create` so I got a 401.

Reply:
The Authorization Header is automatically handled by the `HttpV2Service`, so it should be passed in automatically if you have logged into the web-app with this PR. This actually brings up a good point about calling the new endpoints: We get the Auth token from login, but what do we do if a user is already logged in from a previous session? They're authenticated for the V1 API, and the V2 API tries to cache the Authentication Token, but what if they haven't actually logged in through the web-app since this new change has come in (or they just deleted the Auth Token in browser dev tools)?

Reply:
Ahh that makes sense and is a bit of a problem since most people presumably will still be logged in when we announce legacy planning. Maybe `isLoggedIn` should return the token? Or some other endpoint?

Reply:
I think `isLoggedIn` would make the most sense

Reply:
So, looking at it more: we don't store the token anywhere on the backend and I don't think we should, since that would essentially be going back to storing passwords in the database. To get a token we have to log in to FusionAuth, which requires the user's password, again obviously. I think the right thing to do would be to force re-login, which gets the front end the token. I know we talked about that recently with the "spin forever" bug and there isn't an obvious way to do that in the web-app at the moment, right?